actions.yaml

```diff
@@ -19,3 +19,94 @@ get-service-account:
     - username
     - vhost
   additionalProperties: False
+
```

Comment:
Is the plan to implement the rest of these actions or are they surplus to requirements now?

Reply:
When I wrote the spec I included the actions ```drain-node``` for removing all queue membership from a node and ```migrate-queues```` for replacing one member with a specified alternative all queues. While working on this change those actions started to feel irrelevant for a k8s based deployment where nodes are more transient. I omitted those actions for the moment so it could be discussed further without holding up this PR.